guides/common/modules/proc_retrieving-status-of-services-by-using-web-ui.adoc

```diff
@@ -0,0 +1,18 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="retrieving-the-status-of-services-by-using-web-ui"]
+= Retrieving the status of services by using {ProjectWebUI}
+
+[role="_abstract"]
+{Project} uses a set of back-end services.
+When troubleshooting, you can check the status of {Project} services in {ProjectWebUI}.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Administer* > *About*.
+. In the *System Status* widget, verify the status of system components:
+* On the *{SmartProxies}* tab, view the status of all {SmartProxies}.
+* On the *Available Providers* tab, view the status of enabled compute resource providers.
+* On the *Compute Resources* tab, view the status of attached compute resource providers.
+* On the *Plugins* tab, view the list of installed plugins.
```

Comment:
```suggestion
```

What do you think about this? IMO the system status widget does not provide any value to users that want to check the status of services. I vote to drop these lines.

@aneta-petrova